_includes/header.html

```diff
@@ -1,5 +1,19 @@
 <header>
   <div class="container">
-    <p><a href="/"><img class="fadeimg header-logo" src="{{ '/assets/images/Holochain_header-logo.png' | theme_image_url }}" height="50" alt="Holochain Logo" /></a><span class="text-right no-mobile"><a href="https://github.com/Holochain/holochain-proto">GitHub</a> <a target="_blank" href="https://github.com/Holochain/holochain-proto/blob/whitepaper/holochain.pdf">Whitepaper</a> <a target="_blank" href="https://chat.holochain.org/">Chat</a>  <a href="/team.html">Team</a></span></p>
+    {% if site.lang == "en" %}
+      {% capture lang_prefix %}{% endcapture %}
+    {% else  %}
+      {% capture lang_prefix %}{{site.lang}}/{% endcapture %}
+    {% endif %}
+    <p><a href="/{{lang_prefix}}"><img class="fadeimg header-logo" src="{{ '/assets/images/Holochain_header-logo.png' | theme_image_url }}" height="50" alt="Holochain Logo" /></a><span class="text-right no-mobile">
+      <a href="https://github.com/Holochain/holochain-proto">GitHub</a> <a target="_blank" href="https://github.com/Holochain/holochain-proto/blob/whitepaper/holochain.pdf">Whitepaper</a> <a target="_blank" href="https://chat.holochain.org/">Chat</a>  <a href="/{{lang_prefix}}team.html">Team</a>
+      {% if site.lang == "en" %}
+        {% capture link1 %}{{ site.baseurl_root }}es{{ page.url}}{% endcapture %}
+        <a href="{{ link1 }}" >{% t langs.es %}</a>
+      {% elsif site.lang == "es" %}
+        {% capture link2 %}{{ site.baseurl_root }}en{{ page.url  }}{% endcapture %}
```

Comment:
This actually fails, because it looks like page.url includes the `es` so the resulting link is /es/en/...